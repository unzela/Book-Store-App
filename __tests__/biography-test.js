import 'react-native';
import React from 'react';
import biographyBooks from '../src/data/biography';
import renderer from 'react-test-renderer';

test('Description Snapshot',()=>{
    const snap = renderer.create(
        <biographyBooks/>
    ).toJSON();

    expect(snap).toMatchSnapshot();
});

test('Description Renderer',()=>{
    renderer.create(
        <biographyBooks/>
    );

  
});
