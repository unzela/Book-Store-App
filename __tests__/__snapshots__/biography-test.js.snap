// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description Snapshot 1`] = `<biographyBooks />`;
